cmake/modules/hunter_download_server_url.cmake

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2017 NeroBurner
+# All rights reserved.
+
+include(CMakeParseArguments) # cmake_parse_arguments
+
+include(hunter_status_print)
+include(hunter_internal_error)
+include(hunter_test_string_not_empty)
+
+# used gobal variables
+# - HUNTER_DOWNLOAD_SERVER
+#   - if given, URLs are redirected to the download server
+# - HUNTER_DOWNLOAD_SERVER_NO_REDIRECT
+#   - blacklist specifies server urls, which are not mapped to the download server
```

Comment:
I'm still not happy with "blacklist" term, I think something about "except" or "exclude" is more appropriate.

Reply:
I did miss that one. Will fix